roles/kokonet-setup/handlers/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+- name: restart network
+  service:
```

Comment:
Is this service systemd hosted? If so, you might want to use `systemd` instead of `service`

Reply:
Still now CentOS/Fedora have /etc/init.d/network so I think we should keep it.